docs/sources/getting-started/_index.md

```diff
@@ -1,14 +1,141 @@
 
-1. [Getting Logs Into Loki](get-logs-into-loki/)
-1. [Grafana](grafana/)
-1. [LogCLI](logcli/)
-1. [Labels](labels/)
-1. [Troubleshooting](troubleshooting/)
+This guide assists the reader to create and use a simple Loki cluster.
+The cluster is intended for testing, development, and evaluation;
+it will not meet most production requirements.
+
+The test environment runs the [flog](https://github.com/mingrammer/flog) app to generate log lines.
+Promtail is the test environment's agent (or client) that captures the log lines and pushes them to the Loki cluster through a gateway.
+In a typical environment, the log-generating app and the agent run together, but in locations distinct from the Loki cluster. This guide runs each piece of the test environment locally, in Docker containers.
+
+Grafana provides a way to pose queries against the logs stored in Loki and visualize query results.
+ 
+![Simple scalable deployment test environment](simple-scalable-test-environment.png)
+
+The test environment uses Docker compose to instantiate these parts, each in its own container: 
+
+- One [single scalable deployment](../fundamentals/architecture/deployment-modes/) mode **Loki** instance has:
+    - One Loki read component
+    - One Loki write component
+    - **Minio** is Loki's storage back end in the test environment.
+- The **gateway** receives requests and redirects them to the appropriate container based on the request's URL.
+- **Flog** generates log lines.
+- **Promtail** scrapes the log lines from flog, and pushes them to Loki through the gateway.
+- **Grafana** provides visualization of the log lines captured within Loki.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/install)
+- [Docker Compose](https://docs.docker.com/compose/install)
+
+## Obtain the test environment
+
+1. Create a directory called `evaluate-loki` for the test environment. Make `evaluate-loki` your current working directory:
+    ```bash
+    mkdir evaluate-loki
+    cd evaluate-loki
+    ```
+1. Download `loki-config.yaml`, `promtail-local-config.yaml`, and `docker-compose.yaml`:
+
+    ```bash
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/loki-config.yaml -O loki-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/promtail-local-config.yaml -O promtail-local-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/docker-compose.yaml -O docker-compose.yaml
+    ```
+
+## Deploy the test environment
+
+All shell commands are issued from the `evaluate-loki` directory.
+
+1. With `evaluate-loki` as the current working directory, deploy the test environment using `docker-compose`:
+    ```bash
+    docker-compose up
```

Comment:
This command will aggregate the output of all the containers on the shell, which can be a bit confusing. Maybe it would be better to add the detach flag:
```suggestion
    docker-compose up -d
```